### PR TITLE
PEAR-1983 - Update Case Summary Page for Other Clinical Attributes

### PR DESCRIPTION
**Pull request by amlehman**
## Description
\+ Broke up Diagnoses and FollowUps into their own components 

## Checklist

- [x] Added proper unit tests
- [x] Left proper TODO messages for any remaining tasks
- [x] Scanned for web accessibility with **aXe**, and mitigated or documented
      flagged issues

## Screenshots/Screen Recordings (if Appropriate)
Screenshot 2024-09-10 at 3 32 01 PM
Screenshot 2024-09-10 at 3 32 19 PM

### CODE REVIEW

**Comment by wteouchicago** [comment]
Functionality looks good to me. 👍 

Noticed some styling-related things, which I'll defer to @cemile-ctds:
- I _think_ the "No <node name> Found" text font should be Noto-Sans. Actually, the "No Other Clinical Attributes Found." and "No Molecular Tests Found." text seems to be in a different style from, e.g. "No Family Histories Found." text. I'm not sure if we should make them more similar or something. This case should show the difference: http://localhost:3000/cases/164eeaa7-4229-493f-ae77-0843f36af6b8
- Does the new badge with number in the subtables need to be nearer to the name of the node?

Screenshot 2024-09-11 at 12 14 46 PM
Screenshot 2024-09-11 at 12 44 06 PM

File: packages/portal-proto/src/components/Summary/CategoryTableSummary.tsx [hunk 1]
```diff
@@ -30,13 +30,13 @@ function CategoryTableSummary<TData>({
             withArrow
             withinPortal={false}
           >
-            <ActionIcon variant="subtle">
+            <span className="flex items-center px-1.5">
```
Comment (amlehman) [inline packages/portal-proto/src/components/Summary/CategoryTableSummary.tsx:33]: This is an a11y fix. ActionIcon wraps it in a button which needs a name/label (and button is not the best semantic option for this element).

**Comment by amlehman** [comment]
Hey @cemile-ctds, did you have any feedback on the styling for this ticket?

**Comment by cemile-ctds** [comment]
- The 'No Found' text font should be Noto Sans. 
- We need to ensure consistent font styling for the 'No Found' text. Please refer to the mockup for the correct styling. 
- Additionally, the `<h3>` headings have extra margin. It would be better to remove it.
Screenshot 2024-09-17 at 16 25 29

**Comment by cemile-ctds** [comment]
Looks good on my side.

**Review by craigrbarnes — APPROVED** [review]

**Pull request merged**